.github/workflows/pg-migtests.yml

```diff
@@ -70,57 +86,63 @@ jobs:
         echo "127.0.0.1	yb-tserver-n1" | sudo tee -a /etc/hosts
         psql "postgresql://yugabyte@yb-tserver-n1:5433/yugabyte" -c "SELECT version();"
 
+    - name: "TEST: pg-constraints"
+      run: migtests/scripts/run-test.sh pg/constraints ${{ matrix.use_debezium }}
+    
     - name: "TEST: pg-dvdrental"
-      run: migtests/scripts/run-test.sh pg/dvdrental
+      run: migtests/scripts/run-test.sh pg/dvdrental ${{ matrix.use_debezium }}
 
     - name: "TEST: pg-datatypes"
-      run: migtests/scripts/run-test.sh pg/datatypes
-
-    - name: "TEST: pg-constraints"
-      run: migtests/scripts/run-test.sh pg/constraints
+      run: migtests/scripts/run-test.sh pg/datatypes ${{ matrix.use_debezium }}
 
     - name: "TEST: pg-sequences"
-      run: migtests/scripts/run-test.sh pg/sequences
+      run: migtests/scripts/run-test.sh pg/sequences ${{ matrix.use_debezium }}
 
     - name: "TEST: pg-indexes"
-      run: migtests/scripts/run-test.sh pg/indexes
+      run: migtests/scripts/run-test.sh pg/indexes ${{ matrix.use_debezium }}
 
     - name: "TEST: pg-partitions"
-      run: migtests/scripts/run-test.sh pg/partitions
+      run: migtests/scripts/run-test.sh pg/partitions ${{ matrix.use_debezium }}
+      if: matrix.use_debezium == 0
       
     # Broken for v2.15 and v2.16: https://github.com/yugabyte/yugabyte-db/issues/14529
     # Fixed in 2.17.1.0-b368
     - name: "TEST: pg-partitions-with-index"
-      run: migtests/scripts/run-test.sh pg/partitions-with-indexes
-      if: matrix.version != '2.16.2.0-b41'
+      run: migtests/scripts/run-test.sh pg/partitions-with-indexes ${{ matrix.use_debezium }}
+      if: matrix.version != '2.16.2.0-b41' && matrix.use_debezium == 0
 
     - name: "TEST: pg-views-and-rules"
-      run: migtests/scripts/run-test.sh pg/views-and-rules
+      run: migtests/scripts/run-test.sh pg/views-and-rules ${{ matrix.use_debezium }}
+      if: matrix.use_debezium == 0
     
     - name: "TEST: pg-misc-objects-1 (Types, case-sensitive-table-name, Domain)"
-      run: migtests/scripts/run-test.sh pg/misc-objects-1
+      run: migtests/scripts/run-test.sh pg/misc-objects-1 ${{ matrix.use_debezium }}
+      if: matrix.use_debezium == 0
```

Comment:
For each "disabled" test can you add a jira issue link in a comment? That way, we know which of them to enable when the issue is fixed.

Reply:
for now can we avoid it, i already have a PR in draft which will fix all the issues